user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,464 @@
+:ref:`View page <user-guide-ui-components-view-pages>`.
+
+|
+
+.. image:: ./img/data_management/view/view_history.png
+
+|
+
+This Data Audit is usually used by system administrators, but may also be used by managers in order to see how details 
+of different records were changed, as well as by whom.
+
+
+.. _admin-capabilities-dotmailer-stats:    
+
+Dotmailer Statistic
+-------------------
+
+This capability is meaningful if the system has been integrated with dormatiler. 
+
+Such integration lets users utilize dotmailer as an email campaign engine, as described in the guide on
+:ref:`Integration with Dotmailer <user-guide-dm-integration>`. So, the user can synchronize lists of contacts in OroCRM 
+and dotmailer, and use them to create email campaigns in dotmailer and import them to OroCRM. 
+
+If the *Dotmailer Statistic* capability has been set to *System*, the user will be able to see the dotmailer campaign 
+statistics to analyze the efficiency of the campaign. 
+
+These statistics are usually used by marketing managers and representatives responsible for email campaigns.
+
+.. _admin-capabilities-export-entities:    
+
+Export Entities
+---------------
+
+If this capability is set to *"System"*, the user can export an entity's records into a .CSV file, as 
+described in the :ref:`Import and Export Data <user-guide-export>` guide. 
+
+The :guilabel:`Export` button will appear in the top right corner of the page.
+
+|
+
+.. image:: ./img/roles/export.png 
+
+|
+
+Export is a general productivity tool that is usually enabled for most users.
+
+
+.. _admin-capabilities-general_import:  
+
+General Import/Action Operations
+--------------------------------
+
+This capability enables common operations for import and export, such as the ability to load the error log. It is 
+recommended to set this capability to *"System"* if either *"Export Entities"* or *"Import Entities"* is 
+set to *"System"*. 
+
+
+.. _admin-capabilities-import-entities:    
+
+Import Entities
+---------------
+
+If this capability is set to *"System"*, the user can import records from a .CSV file to OroCRM, as described 
+in the :ref:`Import and Export Data <user-guide-import>` guide. The :guilabel:`Import` button will 
+appear in the top right corner of the page.
+
+|
+
+.. image:: ./img/roles/import.png 
+
+|
+
+This is necessary for users who need to import large sets of data into the system. For example, these may include sales 
+representatives or employees responsible for lead development.
+
+.. _admin-capabilities-jobs:  
+
+Jobs Management
+---------------
+
+If this capability is set to *"System"*, the users can see jobs that have been started in the system, as well as view 
+their current status and their performance log from the *"Job Queue*" and *"Sheduled tasks"* pages. Links to these pages are 
+available in the *"System"* menu.
+
+The *"Job Queue*" and *"Sheduled tasks"* pages are usually usef by system administrators.
```

Comment:
Edit: The *"Job Queue*" and *"Scheduled tasks"* pages are usually used by system administrators.